src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int upperSide;
+    private int lowerSide;
+    private int lateralSide;
+
+    public IsoscelesTrapezoid(int upperSide, int lowerSide, int lateralSide, String color) {
+        this.upperSide = upperSide;
+        this.lowerSide = lowerSide;
+        this.lateralSide = lateralSide;
+        setColor(color);
+    }
+
+    @Override
+    public double getArea() {
+        return ((upperSide + lowerSide) / 2)
+                * Math.sqrt(Math.pow(lateralSide, 2)
+                        - (Math.pow(lowerSide - upperSide, 2) / 4));
+    }
+
+    @Override
+    public String draw() {
+        StringBuilder builder = new StringBuilder();
+        return builder.append("Figure: Isosceles trapezoid, ").append("area: ")
+                .append(Double.valueOf(getArea()).toString()).append(" sq.units, upper side: ")
```

Comment:
use String class for primitive conversation 
```suggestion
                .append(String.valueOf(getArea())).append(" sq.units, upper side: ")
```